LoRaEngine/modules/LoRaWanPktFwdModule/Dockerfile

```diff
@@ -6,16 +6,15 @@ RUN apt-get install -y --no-install-recommends apt-utils build-essential
 RUN git clone https://github.com/Lora-net/packet_forwarder.git
 RUN git clone https://github.com/Lora-net/lora_gateway.git
 RUN sed -i "s|/dev/spidev0.0|/dev/spidev2.0|g" ./lora_gateway/libloragw/src/loragw_spi.native.c
-RUN sed -i "s|IOT_SK_SX1301_RESET_PIN=7|IOT_SK_SX1301_RESET_PIN=5|g" ./lora_gateway/reset_lgw.sh
```

Comment:
If we change this dockerfile that is used by the intel gateway we need to change the corrisponding deployment.template.amd64.json to have the env variable I would also need to test it

Reply:
Done.